### PR TITLE
style(oui-navbar): add style for brand title

### PR DESCRIPTION
**Pull request by AxelPeter**
Linked to
https://github.com/ovh-ux/ovh-ui-angular/pull/183
https://github.com/ovh-ux/ovh-documentation-toolkit/pull/27

### CODE REVIEW

File: packages/oui-navbar/_mixins.less [hunk 1]
```diff
@@ -67,6 +67,15 @@
         color: inherit;
       }
     }
+
+    // Header/Title
+    &__title {
+      display: inline-block;
+      margin-left: rem-calc(15);
+      font-size: rem-calc(28);
+      font-weight: 300;
```
Comment (FredericEspiau) [inline packages/oui-navbar/_mixins.less]: No variables for `15`, `28`, `300` and `#fff` ?

File: packages/oui-navbar/_mixins.less [hunk 2]
```diff
@@ -67,6 +67,15 @@
...
+      color: #fff;
```
Comment (FredericEspiau) [inline packages/oui-navbar/_mixins.less]: I would have used `white` instead 😄

Comment (FredericEspiau) [reply packages/oui-navbar/_mixins.less]: Shorted =! clear

```javascript
setTimeout(blastOff, 86400000);
```

is shorter than

```javascript
const MILLISECONDS_IN_A_DAY = 86400000;

setTimeout(blastOff, MILLISECONDS_IN_A_DAY);
```

but not clearer !

But it's only a detail, I don't really care

File: packages/oui-navbar/_mixins.less (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: Added

File: packages/oui-navbar/_mixins.less (hunk 2 above)
Comment (FredericEspiau) [reply packages/oui-navbar/_mixins.less]: Nevermind, most people seem to agree that `#fff` is better than `white` 😄

**Pull request merged**